BraintreeCore/build.gradle

```diff
@@ -42,22 +42,27 @@ android {
     }
 }
 
+def playServicesWallet = "com.google.android.gms:play-services-wallet:${rootProject.playServicesWalletVersion}"
+
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'androidx.work:work-runtime:2.4.0'
 
     api 'com.braintreepayments.api:browser-switch:2.0.0-beta2'
-    api project(':BraintreeModels')
     api project(':SharedUtils')
 
+    androidTestImplementation playServicesWallet
```

Comment:
This is just for the integration test artifact, so it won't make `play-services-wallet` required for merchants.

Reply:
Had to double-check this! This was the one we needed because of a preliminary assertion we make in the [GooglePayConfiguration](https://github.com/braintree/braintree_android/blob/80d618e610435f25dd610a958bc351a0c6bf13c9/BraintreeCore/src/androidTest/java/com/braintreepayments/api/GooglePayConfigurationTest.java#L25) test. We were able to remove a few others though.